modules.d/01systemd-repart/module-setup.sh

```diff
@@ -35,6 +35,6 @@ install() {
         inst_multiple -H -o \
             /etc/repart.d/*.conf \
```

Comment:
what about this one?

Reply:
There is always that one that gets away :smile:  
Fixed.